templates/README.md

```diff
@@ -14,7 +14,7 @@ The documents declared in this repository are:
 
 ## Contributing
 
-See [contrib-declarations repository](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md) to have explanations on how to add and update services or documents.
+See [contrib-declarations repository](https://github.com/ambanum/OpenTermsArchive/blob/main/CONTRIBUTING.md) to have explanations on how to add and update services or documents.
```

Comment:
The text content of the link is obsolete and needs an update too 😉

Reply:
Link is obsolete but working and the other is a 404.
Documentation website does not hold this info yet and as I do not know when it will be merged, I'd rather update this.

Reply:
Sorry if I wasn't clear enough.

```suggestion
To add or update documents, see the [reference documentation](https://github.com/ambanum/OpenTermsArchive/blob/main/CONTRIBUTING.md).
```